cmd/google_cloud_ops_agent_engine/main.go

```diff
@@ -65,7 +70,10 @@ func run() error {
 	log.Printf("Built-in config:\n%s", apps.BuiltInConfStructs["linux"])
 	log.Printf("Merged config:\n%s", uc)
 
-	runStartupChecks(*service)
+	if *service == "" {
```

Comment:
Is there a way to only have one entry point for the `runHealthChecks` that works for either the Ops Agent startup or just the `--healthchecks` flag.

Reply:
I think this was addressed.